docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -808,20 +805,18 @@ The acceptable flags include all those of `kubectl` except for
 - `-X`: turn on debug echoing of the commands inside the script that
   implements this command.
 
+The space manager is involved in the creation of the workload management
+space. To access the space manager, the following environment variables
+are used: SM_CONFIG specifies the path to the space manager kubeconfig 
+file. SM_CONTEXT specifies the context of the space manager within the 
+kubeconfig file. IN_CLUSTER specifies whether the space manager is
+accessed from within the hosting cluster or externally. Note that 
```

Comment:
By not mentioning _who_ is doing the access, this text currently addresses _all_ access. I think you do not mean that, you mean to talk about the access done by this command. Please make the text that specific.